lib/presets.js

```diff
@@ -428,6 +428,63 @@ module.exports = {
           return callback(error);
         }
       });
+    },
+
+    /**
+     * @summary Update quoted version immediately following a regex
+     * @function
+     * @public
+     *
+     * @param {Object} options - options
+     * @param {String} [options.baseDir] - directory to prefix onto options.file
```

Comment:
> Since we are going to use `baseDir` as a generic parameter for all presets, I think we need a more generic description than directory to prefix onto options.file.

Good point - not all presets will have a `file` option. What about "relative directory to append to cwd" ?

Reply:
Yes. Later on when `baseDir` is added globally its purpose could be clarified in the README.